src/main/java/uk/gov/hmcts/cmc/claimstore/events/DocumentReadyToPrintEvent.java

```diff
@@ -9,12 +9,19 @@
     private final Claim claim;
     private final Document defendantLetterDocument;
     private final Document sealedClaimDocument;
+    private final String authorisation;
 
-    public DocumentReadyToPrintEvent(Claim claim, Document defendantLetterDocument, Document sealedClaimDocument) {
```

Comment:
Looks like a good place for lombok to shine

Reply:
used lombok @Data